builder/googlecompute/driver_gce.go

```diff
@@ -140,6 +149,15 @@ func NewDriverGCE(config GCEDriverConfig) (Driver, error) {
 		return nil, err
 	}
 
+	if metadata.OnGCE() {
+		log.Printf("[INFO] On GCE, capture service account for OSLogin...")
+		thisGCEUser, err = metadata.NewClient(&http.Client{}).Email("")
```

Comment:
This change causes a timeout on the official gitlab ci runners that are running in the GCP and are supposed to build a image with type of googlecompute.

```
packer build -force packer/runner.json
googlecompute: output will be in this color.
Build 'googlecompute' errored after 30 seconds 2 milliseconds: Get "http://169.254.169.254/computeMetadata/v1/instance/service-accounts/default/email": dial tcp 169.254.169.254:80: i/o timeout
==> Wait completed after 30 seconds 2 milliseconds
```